rousseau_vote/l10n/it.json

```diff
@@ -44,9 +44,10 @@
     "vote-limit": "Hai già selezionato il numero massimo di opzioni consentite per questa votazione",
     "vote-complete": "Il tuo voto è stato acquisito correttamente",
 
-    "vote-preferences-1": "Hai ",
+    "vote-preferences-1": "Hai ancora ",
```

Comment:
We need more intuitive names. What's that a message, badge, alert? Plus, you should consider [sprintf](https://pub.dev/packages/sprintf) for cleaner code. This can become something like:
```
"vote-preferences-message-plural": "Hai ancora %s preferenze a disposizione",
"vote-preferences-message-singular": "Hai ancora 1 preferenza a disposizione",
```

Reply:
Ok I will consider using sprintf in the next occasions.
```
TextSpan(
        text: remainingPreferencesText1,
        style: const TextStyle(fontSize: 18),
        children:<TextSpan>[
          TextSpan(
            text: remainingPreferences.toString(),
            style: TextStyle(
              fontSize: 28,
              fontWeight: FontWeight.w600
            )
          ),
          TextSpan(
            text: remainingPreferencesText2
          )
        ],
      

)
```

This can't be applied here though because I made use of different font sizes for different TextSpan, therefore the need of two separate strings.